content/concepts/dolt/rdbms/replication.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Replication
+---
+
+# Replication
+
+## What is a Replication?
+
+Replication is the ability for an RDBMS to synchronize a master server with one or more read replicas. In this configuration, the master database serves reads and writes while the replicas serve just reads.
+
+## How to use Replication
+
+Replication is used for disaster recovery and to increase read throughput by distributing read load.
+
+For the disaster recovery, if your master server goes offline, your database can still serve read traffic from its replicas. Often a manual or automated process can elect and configure a replica to be the master instance limiting downtime.
+
+To increase read throughput, multiple replicas can be used to scale reads horizontally. If you have N replicas and your master still takes reads, each read replica serves 1/N+1 percent of the read traffic.
```

Comment:
Should we mention anything about how to use replicas from your application? Maybe just acknowledging that application code need to be aware of the replicas in order to route requests there, so people don't think replication will reroute requests automatically?